config/jest/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   testPathIgnorePatterns: ['<rootDir>[/\\\\](dist|node_modules)[/\\\\]'],
   moduleNameMapper: {
-    '\(seek-style-guide\/react|\.(css|less)$)': require.resolve(
+    '(seek-style-guide/react|.(css|less)$)': require.resolve(
```

Comment:
Testing came back as below.
`new RegExp('a.').exec('aa')` => [ 'aa', index: 0, input: 'aa' ]
`new RegExp('a\.').exec('aa')` => [ 'aa', index: 0, input: 'aa' ]
`new RegExp('a\\.').exec('aa')` => null

From this I believe that the `\.` in the above code was always resolving to `.`, and was checking for any character rather than a literal `.`.